.github/workflows/run_integration_tests.yml

```diff
@@ -138,21 +139,23 @@ jobs:
           }
 
           Write-Host "Run tests"
-          $parallelXunitOption = "none" 
 
           # Test parallelization is disabled until we can solve concurrent dotnet publish issues with ConsoleMF usage
+          $json = Get-Content "${{ env.integration_tests_path }}/xunit.runner.json" | ConvertFrom-Json
+          $json | Add-Member -Name "parallelizeAssembly" -Value $false -MemberType NoteProperty
+          $json | Add-Member -Name "parallelizeTestCollections" -Value $false -MemberType NoteProperty
           # if ("${{ matrix.namespace }}" -like "Logging.*" ) {
-          #   Write-Host "Logging tests detected, running xunit tests in parallel."
-          #   $parallelXunitOption = "all"
+          #   $json.parallelizeAssembly = $true
```

Comment:
Looks like some stray commented-out lines

Reply:
No, never mind, that's for the logging parallelization which we want to leave commented-out.  My bad.